src/main/java/net/atos/zac/flowable/cmmn/CompleteTaskInterceptor.java

```diff
@@ -5,13 +5,24 @@
 
 package net.atos.zac.flowable.cmmn;
 
+import static java.lang.String.format;
+import static net.atos.zac.configuratie.ConfiguratieService.STATUSTYPE_OMSCHRIJVING_INTAKE;
+
+import java.util.UUID;
+import java.util.logging.Logger;
+
+import org.flowable.cmmn.api.runtime.PlanItemInstance;
 import org.flowable.cmmn.engine.CmmnEngineConfiguration;
 import org.flowable.cmmn.engine.impl.interceptor.DefaultCmmnIdentityLinkInterceptor;
 import org.flowable.task.service.impl.persistence.entity.TaskEntity;
 
+import net.atos.client.zgw.zrc.model.Zaak;
 import net.atos.zac.flowable.FlowableHelper;
 
 public class CompleteTaskInterceptor extends DefaultCmmnIdentityLinkInterceptor {
+    private static final Logger LOG = Logger.getLogger(CompleteTaskInterceptor.class.getName());
+    private static final String STATUS_TOELICHTING = "Status gewijzigd";
```

Comment:
I would prefer us retrieving these descriptions from a service, instead of defining these values throughout the code. Is this something on our radar?

Reply:
nope